torch/cuda/__init__.py

```diff
@@ -287,6 +299,19 @@ def get_device_capability(device):
         return torch._C._cuda_getDeviceCapability(device)
 
 
+def get_total_memory(device):
```

Comment:
I put that comment in another PR too. Can we just expose the device properties struct?